robots/cmd/per-test-execution/main.go

```diff
@@ -89,10 +104,21 @@ func (o options) loadDefaults() error {
 		}
 		opts.K8sVersion = k8sStableReleaseVersionRegex.FindAllStringSubmatch(k8sStableReleaseVersion, -1)[0][1]
 	}
+	if o.outputDirectory != defaultOutputDirectory {
```

Comment:
This appears to be more of a validation, would it be better under validate()?

Reply:
Fair - updated!